CPU-POOLS.md

```diff
@@ -301,18 +301,21 @@ files and look like this:
 
 ```
      cpupools:
-       - name: "reserved"
-         cpus: "0-7,9"
+       - name: "ml"
+         cpus: "4-8,10"
+       - name: "avx512"
+         cpus: "*"
        - name: "default"
-         cpus: "8,10-12"
+         cpus: "@2"
```

Comment:
Hmm... since we're practically doing translation between configurations anyway, maybe this could be made more user-friendly by using something like numCpus: "2", or cpuCount: "2" here, which then would be translated to "@2" for the backend.

Reply:
I think we are moving to unify the kubelet config file syntax and pool-tool profile file syntax soon, to avoid misunderstandings and to make it easier to test that the expected configuration is correct. Also, I'm hoping to use the same configuration validation code in both places.